test/run_test.py

```diff
@@ -33,6 +33,7 @@
     'test_import_time',
     'test_public_bindings',
     'test_type_hints',
+    'test_ao',
```

Comment:
Should this be test_ao_sparsity instead?

Reply:
This is weird -- we changed it with @kazhou, weirdly it is still here.